### ISSUE TITLE
0.14.2 Mod Custom Block Texture Problem. Just Gray

### ISSUE THREAD

**Issue by TheMCProf** [zhuowei/MCPELauncher#990]
When I downloaded mods with custom blocks and textures. When I import the .js and texture .zip into Block Laucher, there isn't a problem;. But when I place the custom block down, the texture is just gray but there isn't any error pop-ups when I place it down. This has happened with so many mods. 
By the way, some of the mods are 0.14.0/1 but are compatible with this version.
Please iron out these bugs. Thanks!

**Comment by Black-DeathPE** [comment]
Yep, this is a unsurprising MCPE 0.14.2 bug with render type 4, which modders use to allow proper block transparency.

**Comment by alexplaysgames6992** [comment]
Mine is also the same put when I try to put a texturepack it goes back to mcpe homepage and I'm on lollipop
